playbooks/clouds.yaml.j2

```diff
@@ -0,0 +1,54 @@
+clouds:
```

Comment:
Document start missing.

Reply:
Genuine question: what is the benefit? It's not necessary, right?

Reply:
Yamllint is not happy when it is missing. That's why we put it everywhere.